### DISCUSSION TITLE
i have a problem when compiling dosrznrc input

### DISCUSSION THREAD

**Discussion by morfius84** [nrc-cnrc/EGSnrc#1003] [General]
I have a problem when compiling dosrznrc input

```
EXECUTING  : mingw32-make opt EGS_CONFIG=D:\EGSnrc\EGSnrc-master\HEN_HOUSE\specs\win3264.conf
fatal: not a git repository (or any of the parent directories): .git
fatal: not a git repository (or any of the parent directories): .git
.
.
"Fortran compiling dosrznrc_win3264.F using flags '-cpp  -fPIC -O2 -mtune=native -DCOMPILE_TIME="\"Mon 06/05/2023 10:28 PM Egypt Standard Time\"" ' and extra objects/libs 'D:\EGSnrc\EGSnrc-master\HEN_HOUSE\lib\win3264\egs_c_utils.obj D:\EGSnrc\EGSnrc-master\HEN_HOUSE\lib\win3264\load_beamlib.obj   '"
dosrznrc_win3264.F:17250:33:

17250 |       write(i_log,'(a)') GIT_HASH
      |                                 1
Error: Symbol 'git_hash' at (1) has no IMPLICIT type
mingw32-make[1]: *** [D:\EGSnrc\EGSnrc-master\HEN_HOUSE\makefiles\standard_makefile:90: D:\EGSnrc\EGSnrc-master\third-party\bin\win3264\dosrznrc.exe] Error 1
mingw32-make: *** [D:\EGSnrc\EGSnrc-master\HEN_HOUSE\makefiles\standard_makefile:189: opt
```

**Comment by rtownson** [comment]
Is this fixed by https://github.com/nrc-cnrc/EGSnrc/pull/972 ?

**Comment by rtownson** [comment]
By the way, please install `git` if you haven't already. It will make your life working with EGSnrc much easier and more reliable.

https://gitforwindows.org/

**Reply by morfius84** [reply]
Thx very much I worked on your advice

**Reply by morfius84** [reply]
i tried to reinstall EGsnrc2023 and the problen is still appear sorry iam not a programmer iam physist

**Reply by morfius84** [reply]
plz can you find solution

**Reply by rtownson** [reply]
Did you install git? If you do this, then remove your old EGSnrc installation. Then try installing again, using git clone. The instructions for this (`option 1`) are here:

https://github.com/nrc-cnrc/EGSnrc/wiki/Install-EGSnrc-on-Windows#option-1-clone-the-git-repository-strongly-recommended

Let me know if this solves the issue.
